javascripts/discourse/initializers/dc-homepage.js.es6

```diff
@@ -0,0 +1,14 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "dc-homepage",
+  initialize() {
+    withPluginApi("0.8", api => {
+      api.modifyClass("route:discovery-categories", {
+        findCategories() {
+          return this._findCategoriesAndTopics("latest");
```

Comment:
@duranmla this works, but can be optimized later to only fetch the topics we need, probably it will reduce the load time of the home page

Reply:
I guess that's where the idea of customisation from the backend side would be useful. But yes, we can iterate further.